proto/v1/vulnerability.proto

```diff
@@ -147,6 +147,11 @@ message VulnerabilityNote {
   // upstream timestamp from the underlying information source - e.g. Ubuntu
   // security tracker.
   google.protobuf.Timestamp source_update_time = 6;
+
+  // Output only. CVSS version used to populate cvss_score and severity.
```

Comment:
Why output only? On the note we store/populate it.

Reply:
Done.